config/config.exs

```diff
@@ -45,6 +45,9 @@ config :archethic, :mut_dir, "data"
 
 config :archethic, :marker, "-=%=-=%=-=%=-"
 
+# size represents in mb i.e 1024 * 1024  * 3 bytes
+config :archethic, :transaction_data_content_max_size, 3
```

Comment:
What do you think of putting the number in bytes instead of megabytes ? 
This will avoid the division in the changeset to convert bytes to mb .